src/rt/store.rs

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Is this file still used? I don't see a `mod store` in `rt/mod.rs`; and it looks like the `Store` type now lives in `rt/object.rs`.

Reply:
good catch :+1: